_posts/2020-11-01-Fortran-Newsletter-November-2020.md

```diff
@@ -0,0 +1,113 @@
+mpifx, NTPoly, NWChem, OpenMolcas, PoisFFT, QMD-PROGRESS, scalapackfx,
+tapenade, wannier90, and xtb.
+* [#145](https://github.com/fortran-lang/fortran-lang.org/pull/145),
+[#146](https://github.com/fortran-lang/fortran-lang.org/pull/146),
+[#154](https://github.com/fortran-lang/fortran-lang.org/pull/154),
+[#158](https://github.com/fortran-lang/fortran-lang.org/pull/158):
+Minor fixes and improvements.
+
+Ongoing work:
+
+* [#160](https://github.com/fortran-lang/fortran-lang.org/pull/160) (WIP):
+In-depth introduction for Fortran with Make.
+* [#156](https://github.com/fortran-lang/fortran-lang.org/pull/156) (WIP):
+Updating the mini-book on building programs.
+
+[Let us know](https://github.com/fortran-lang/fortran-lang.org/issues) if you have any suggestions for the website and its content.
+We welcome any new contributors to the website and the tutorials page in particular - see the [contributor guide](https://github.com/fortran-lang/fortran-lang.org/blob/master/CONTRIBUTING.md) for how to get started.
+
+## Fortran Standard Library
+
+TODO What's new
+
+The candidate for file system operations to be included in stdlib is being developed by
+[@MarDiehl](https://github.com/MarDiehl) and [@arjenmarkus](https://github.com/arjenmarkus)
+in [this repository](https://github.com/MarDiehl/stdlib_os).
+Please try it out and let us know how it works, if there are any issues, or if the API can be improved.
+
+## Fortran Package Manager
+
+TODO What's new
+
+fpm is still in early development and we need as much help as we can get.
+Here's how you can help today:
+
+* Try to use it. Does it work? No? Let us know! Read the [fpm packaging guide](https://github.com/fortran-lang/fpm/blob/master/PACKAGING.md) to learn how to build your package with fpm.
+* Browse the [open issues](https://github.com/fortran-lang/fpm/issues) and see if you can help implement any fixes or features.
+* Adapt your Fortran package for fpm and submit it to the [Registry](https://github.com/fortran-lang/fpm-registry).
+* Improve the documentation.
+
+The short-term goal of fpm is to make development and installation of Fortran packages with dependencies easier.
+Its long term goal is to build a rich and decentralized ecosystem of Fortran packages and create a healthy
+environment in which new open source Fortran projects are created and published with ease.
+
+## Compilers
+
+### Classic Flang
+
+TODO @gklimowicz
+
+### LLVM Flang
+
+TODO @gklimowicz
+
+### LFortran
+
+TODO @certik
+
+You can follow LFortran on Twitter for latest updates: [@lfortranorg](https://twitter.com/lfortranorg).
+
+## Events
+
+* We had our 5th Fortran Monthly call on October 27.
+You can watch the recording below:
+
+TODO update Youtube embed when the video is up
+
+<!--<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/Ei6CAZblOro" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>-->
```

Comment:
<!--<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/om869cZHeRU" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>-->